Gopkg.toml

```diff
@@ -84,7 +85,7 @@
 
 [[override]]
   name = "github.com/mattn/go-isatty"
-  revision = "281032e84ae07510239465db46bf442aa44b953a"
+  revision = "3fb116b820352b7f0c281308a4d6250c22d94e27"
```

Comment:
previous got outdated? not that important just want to know reason of the change

Reply:
Yes. It was updated by go-ethereum too.